README.md

```diff
@@ -303,10 +303,10 @@ Habushu will replace invocations of Poetry's `build` and `publish` commands with
 `poetry-monorepo-dependency-plugin`, which are `build-rewrite-path-deps` and `publish-rewrite-path-deps`, respectively.
 
 To further ease working with monorepo dependencies, Habushu will specially handle how these dependencies are written to 
-intermediate `requirements.txt` exports if they are included in the `[tool.poetry.group.monorepo.dependencies]` group 
-within your `pyproject.toml` file.  In this case, the dependencies will be excluded from the export unless `rewriteLocalPathDepsInArchives`
-is set to true.  This feature can be useful if installing the exported `requirements.txt` on another machine and/or 
-Docker container.  If a monorepo dependency is not included in this specific group, it will be exported with a local path.
+intermediate `requirements.txt` exports.  In this case, the dependencies will be excluded from the export unless 
+`rewriteLocalPathDepsInArchives` is set to true.  This feature can be useful if installing the exported 
+`requirements.txt` on another machine and/or Docker container.  If a monorepo dependency is not included in this 
+specific group, it will be exported with a local path.
```

Comment:
A: This last sentence is a non-sequitur/inaccurate?

Reply:
Good catch!  ✅ fixed